woocommerce/navigation
# Migrate remaining WooCommerce menu items

Fixes #9 

This PR migrates any remaining items under the original "WooCommerce" parent item to the new "Settings" menu.

Note that this also migrates some items like "Orders" and "Coupons" but will be fixed by https://github.com/woocommerce/navigation/issues/8.

### Screenshots
<img width="593" alt="Screen Shot 2020-07-07 at 10 32 52 AM" src="https://user-images.githubusercontent.com/10561050/86738152-81b6cb80-c03d-11ea-81c8-6aa7bb180140.png">
<img width="198" alt="Screen Shot 2020-07-07 at 10 32 18 AM" src="https://user-images.githubusercontent.com/10561050/86738169-854a5280-c03d-11ea-9bb7-7f59d3f53e96.png">

<img width="329" alt="ScreenCapture at Thu Jul 9 12:33:04 EDT 2020" src="https://user-images.githubusercontent.com/2098816/87066424-8d8ec300-c1e0-11ea-9c8e-6c585a5caabd.png">


### Testing

1. Open up the WP dashboard to a non-wc page.
1. Make sure the WooCommerce menu is shown but no submenu is shown.
1. Optionally, check the source code to make sure that no submenu items exist (note that we hide the submenu instead of unsetting to avoid breaking overriding this link as WCA does).
1. Open a WC registered screen, such as settings.  `wp-admin/admin.php?page=wc-settings`
1. Make sure items that previously remained under the WooCommerce menu item now exist in the navigation global under the children in "settings."  `window.wcSettings.wcNavigation[3].children`
